src/coin_test/analysis/build_datapane.py

```diff
@@ -0,0 +1,105 @@
+"""Functions to build Datapane Locally."""
+
+from typing import Sequence
+
+import datapane as dp
+
+from .graphs import (
+    _get_strategy_results,
+    ConfidenceDataPlot,
+    ConfidencePricePlot,
+    ConfidenceReturnsPlot,
+    PlotParameters,
+    ReturnsHeatmapPlot,
+    SignalPricePlot,
+)
+from .tables import SummaryTearSheet, TearSheet
+from ..backtest import BacktestResults
+
+
+def _build_strategy_page(
+    strategy_name: str,
+    results: Sequence[BacktestResults],
+    plot_params: PlotParameters,
+) -> dp.Page:
+    tear_sheet = TearSheet.create(results)
+
+    confidence_price = ConfidencePricePlot.create(results, plot_params)
+    confidence_returns = ConfidenceReturnsPlot.create(results, plot_params)
+    returns_heatmap = ReturnsHeatmapPlot.create(results, plot_params)
+    signal_price = SignalPricePlot.create(results, plot_params)
+
+    blocks = [
+        "# " + strategy_name,
+        "## Tables",
+        "### Tear Sheet",
+        tear_sheet,
+        "## Graphs",
+        "### Portfolio Value Over Time",
+        confidence_price,
+        "### Portfolio Return Over Time",
+        confidence_returns,
+        "### Portfolio Returns vs Dataset Returns",
+        returns_heatmap,
+        "### Signal Plot",
+        signal_price,
+    ]
+    page = dp.Page(
+        title=strategy_name,
+        blocks=blocks,
+    )
+    return page
+
+
+def _build_strategy_pages(
+    results: Sequence[BacktestResults], plot_params: PlotParameters
+) -> list[dp.Page]:
```

Comment:
Again docstrings for these functions? Not convinced we need them but seems helpful